vochain/scrutinizer/scrutinizer_test.go

```diff
@@ -84,20 +91,38 @@ func testProcessList(t *testing.T, procsCount int) {
 
 	// Add 10 entities and process for storing random content
 	for i := 0; i < 10; i++ {
-		sc.addEntity(util.RandomBytes(20), util.RandomBytes(32))
+		pid := util.RandomBytes(32)
+		if err := state.AddProcess(&models.Process{
+			ProcessId:   pid,
+			EntityId:    util.RandomBytes(20),
+			VoteOptions: &models.ProcessVoteOptions{MaxCount: 8, MaxValue: 3},
+		}); err != nil {
+			t.Fatal(err)
```

Comment:
might as well use qt.Assert everywhere? :)

Reply:
Actually a full qt.Assert refactor needs to be performed on this test file. I only added qt.Assert to the new lines to not change everything on a commit which is not directly related with this refactor (note that this snippet is a copy&paste from an old snippet). But sure, I could change it anyway on this specific case.